src/shadowbox/model/access_key.ts

```diff
@@ -59,6 +59,8 @@ export interface AccessKeyRepository {
   listAccessKeys(): AccessKey[];
   // Changes the port for new access keys.
   setPortForNewAccessKeys(port: number): Promise<void>;
+  // Changes the hostname for new access keys.
+  setHostname(hostname: string): void;
```

Comment:
I think it's less ambiguous to use `host` instead of `hostname`.  However, I don't feel very strongly about it, since it is also common to use `hostname` to refer to both domain names and IPs. Agreed that we should do the change in a consistent and backward compatible way.